### ISSUE TITLE
Wiki instructions no longer valid

### ISSUE THREAD

**Issue by cryptosig** [cryptonomex/graphene#678]
The directions where to download QT 5.5 is no longer valid in the wiki.

http://download.qt.io/official_releases/qt/5.5/5.5.0/

is no longer stored by that server, next closest version would be

http://download.qt.io/official_releases/qt/5.6/5.6.0/single/qt-everywhere-opensource-src-5.6.0.tar.gz

**Comment by abitmore** [comment]
Actually QT is no longer needed.

The main BitShares repo has been moved to https://github.com/bitshares/bitshares-core, the wiki here is not updated.

**Comment by cryptosig** [comment]
Right, but these instructions are linked for the peerplays readme. So perhaps that readme needs to point to an updated wiki, or something, so people with less skill can compile without difficulties.

**Comment by vikramrajkumar** [comment]
The `cryptonomex` repos are no longer maintained. Unless someone takes responsibility for managing them, I would update the Peerplays repos directly and/or merge updates to and from the `bitshares` repos.
